core/modules/simpletest/tests/icon.test

```diff
@@ -129,6 +129,16 @@ class IconTestCase extends BackdropWebTestCase {
     $expected_svg = '<svg xmlns="http://www.w3.org/2000/svg" width="16" height="16" version="1.1" viewBox="0 0 16 16" aria-hidden="true"><circle cx="8" cy="8" r="4"/></svg>';
     $updated_svg = image_add_svg_attributes($initial_svg, $options);
     $this->assertEqual($updated_svg, $expected_svg, 'Title tag stripped and aria attribute added properly.');
+
+    // Make sure, a comment does not break attribute handling.
+    $initial_svg = '<!-- Some vendor comment --><svg width="16" height="16" version="1.1" viewBox="0 0 16 16" xmlns="http://www.w3.org/2000/svg"><circle cx="8" cy="8" r="4"></circle></svg>';
```

Comment:
That's good thinking here 👍🏼 ...can we please add another test, where the comment comes right after the `<svg>` opening tag? That would also make sure that things don't blow up if `firstChild` is used instead of `firstElementChild`.

Reply:
> That's good thinking here 👍🏼 ...can we please add another test, where the comment comes right after the `<svg>` opening tag? That would also make sure that things don't blow up if `firstChild` is used instead of `firstElementChild`.

Actually there is no danger of that now, since the PR uses this, you can rest assured that you will always get the svg element:
`$svg_tag = $doc->getElementsByTagName('svg')->item(0);`

Reply:
Oh, I misread @klonos's comment. Actually, good idea.